src/coreclr/vm/assemblyspec.cpp

```diff
@@ -1406,11 +1382,8 @@ BOOL AssemblySpecBindingCache::StoreException(AssemblySpec *pSpec, Exception* pE
         pBinderToSaveException = pSpec->GetBinder();
         if (pBinderToSaveException == NULL)
         {
-            if (!pSpec->IsAssemblySpecForCoreLib())
```

Comment:
Is `IsAssemblySpecForCoreLib` still used or can we remove it?

Reply:
Yep, `IsAssemblySpecForCoreLib` can be removed.